tex/set-theory/models.tex

```diff
@@ -693,9 +716,15 @@ \section{Picturing inner models}
 \end{dproblem}
 
 \begin{sproblem}
-	[Inaccessible cardinals produce models]
+	[Strongly inaccessible cardinals produce models]
```

Comment:
Definition of page 590 in appendix A of Chang, Keisler of "inaccessible" is the same as our "strongly inaccessible".

That's probably the reason for the confusion here (in the Napkin, "inaccessible" was never defined, and given "weakly inaccessible" and "strongly inaccessible" *are* defined (with "strongly inaccessible" being a stronger property than "weakly inaccessible"), a natural assumption would be "inaccessible = weakly inaccessible OR strongly inaccessible = weakly inaccessible", which doesn't seem to hold.